storm-server/src/test/java/org/apache/storm/LocalStateTest.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.storm;
```

Comment:
You need to add the Apache license to the top of this file. You can copy it from any of the other files in this directory.